src/ts/createView.tsx

```diff
@@ -59,11 +59,11 @@ const createView = ({
         commands.applySuggestions(suggestionOpts);
         commands.stopHover();
       }}
-      onIgnoreMatch={
-        onIgnoreMatch &&
+      onMarkCorrect={
+        onMarkCorrect &&
         (match => {
           commands.ignoreMatch(match.matchId);
-          onIgnoreMatch(match);
+          onMarkCorrect(match);
```

Comment:
There is now a difference between the callback name and the command. I think this is acceptable but if anyone feels differently, please let me know.